web/html/src/components/input/form-utils.ts

```diff
@@ -76,6 +78,7 @@ export function unflattenModel<T>(flatModel: Record<string, T>): TreeLikeModel<T
 
   Object.keys(flatModel)
     .sort()
+    .filter(name => !stripBlankValues || ((flatModel[name] || "") !== ""))
```

Comment:
I would recommend extracting this part into a separate function and keep the signature for `unflattenModel` as is. That allows us to keep a fairly clean separation of concerns, where one function does one thing.

Reply:
OK